NN_module/Makefile

```diff
@@ -21,7 +22,7 @@ LDFLAGS = $(shell nf-config --flibs)
 
 
 PROGRAM = test
-SRC = nn_cf_net.f90 nn_convection_flux.f90 nn_interface_SAM.f90 test.f90
+SRC = precision.f90 test_utils.f90 nn_cf_net.f90 nn_convection_flux.f90 nn_interface_SAM.f90 test.f90
 OBJECTS = $(SRC:.f90=.o)
 
 
```

Comment:
Can we put a comment in the readme about running the tests via `make test` (and that this requires `ifort`). I do have an alternate makefile for gfortran if helpful?

Reply:
I have made the Makefile clearer about running with ifort or gfortran and 2 ppossible options.
Is that OK, or do you want me to also add to the readme?

Reply:
I have fixed the readme to reflect the options for different compilers.

Reply:
super